addons/pos_restaurant/static/src/overrides/models/models.js

```diff
@@ -42,6 +42,13 @@ patch(Order.prototype, {
     setCustomerCount(count) {
         this.customerCount = Math.max(count, 0);
     },
+    amountPerGuest(numCustomers = this.customerCount) {
+        if (numCustomers === 0) {
+            return this.env.utils.formatCurrency(0);
+        }
+        const totalDue = this.get_total_with_tax() + this.get_rounding_applied();
+        return this.env.utils.formatCurrency(totalDue / numCustomers);
```

Comment:
in general your methods should return the numeric value and the you would format in the template. This way you can use that value in other computations too